api/serverpb/server.proto

```diff
@@ -290,18 +292,28 @@ message PlatformSignUpRequest {
 
 message PlatformSignUpResponse {}
 
-message PlatformSignInRequest {
```

Comment:
That was part of public API and can be used by third-party scripts, etc. I assume we can't just drop this stuff, but we can deprecate it. 

@BupycHuk  correct me if I'm wrong.

Reply:
Yeah, that's correct.

Reply:
And so, we can revert changes in api.descriptor

Reply:
Ok, to make it clear, you want the singin api not to be removed. So instead of replacing it, we just add a new one. @artemgavrilov @BupycHuk

Reply:
yes, and add `[deprecated]` to old ones.